gbpservice/nfp/orchestrator/config_drivers/heat_driver.py

```diff
@@ -676,7 +676,7 @@ def _update_node_config(self, auth_token, tenant_id, service_profile,
             LOG.error(_LE("No provider cidr availabale"))
             return None, None
```

Comment:
Don't see delete of orchestration drivers in this PR. Aren't they is separate files?